src/components/ItemDisplaySelect/ItemDisplaySelect.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react'; // eslint-disable-line no-unused-vars, prop-types
+import AssignmentIcon from '@material-ui/icons/Assignment';
+import PropTypes from 'prop-types';
+import AppsIcon from '@material-ui/icons/Apps';
+import MenuIcon from '@material-ui/icons/Menu';
+import MuiCheckbox from '@material-ui/core/Checkbox';
+import { withStyles } from '@material-ui/core';
+
+export const DisplayCardType = Object.freeze({
+  List: 'DISPLAY_CARD_TYPE_LIST',
+  SmallCard: 'DISPLAY_CARD_TYPE_SMALL_CARD',
+  BigCard: 'DISPLAY_CARD_TYPE_BIG_CARD',
+});
+
+const styles = {
+  buttonBox: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+  checkboxRoot: {
+    padding: 8,
+    margin: 6,
+  },
+};
+
+const prepareDisplayTypes = displayType => {
```

Comment:
Maybe not as intuitive, but you could write this function like this:

```suggestion
const prepareDisplayTypes = displayType => {
  return [
    displayType === DisplayCardType.List,
    displayType === DisplayCardType.SmallCard,
    displayType === DisplayCardType.BigCard,
  ];
};
```

Reply:
I love this improvement!